src/main/scala/com/typesafe/sbt/packager/debian/Keys.scala

```diff
@@ -18,6 +18,8 @@ trait DebianKeys {
   // Package building
 
   val debianControlFile = TaskKey[File]("debian-control-file", "Makes the debian package control file.")
+
+  @deprecated("Use generic maintainerScript task instead", "1.0.3")
```

Comment:
Yes. However this setting is meant to be used by every packaging system that provides `maintainerScripts`. I will probably push this setting up to the universal keys.